04-scheduling.egg

```diff
@@ -197,18 +197,36 @@
 ;;
 ;; Currently, `egglog-experimental` implements one scheduler, `back-off`. The idea of `back-off` is that it will ban
```

Comment:
not about your diff, but maybe add a forward pointer that says in section 6 we will implement our own scheduler.